spec/videos/video_spec.rb

```diff
@@ -87,4 +87,77 @@
       it { expect { video }.to raise_error(Funky::ConnectionError) }
     end
   end
+
+  describe '.find_by(url: url)' do
+    let(:video) {Funky::Video.find_by(url: url)}
+
+    context "given an existing video with this url format: 'https://www.facebook.com/{account_name}/videos/vb.{alt_account_id}/{video_id}/'" do
+      let :url do
+        'https://www.facebook.com/FOXDanmark/videos/vb.129316923794451/965037490222386/'
+      end
+
+      it { expect(video.id).to be_a(String) }
+      it { expect(video.like_count).to be_an(Integer) }
```

Comment:
In all these tests, we don't really care about `like count` being an Integer and so on.

We care about `id` being `965037490222386` (and not, say `129316923794451`).

This PR is just about a "string parser" that knows nothing else about Facebook data.

I would even have all these tests in a different spec file